src/index.less

```diff
@@ -1,6 +1,7 @@
 body {
   position: relative;
   min-height: 100vh;
+  overflow-y: scroll;
```

Comment:
This is to prevent page jumping few pixels in width. Having results might force scrollbars while loading state does not (list is very short). Setting vertical scrollbar to be always visible means no changes in document width as these two state change.